app/script.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+$(function () {
+
+	//currently, there's a bug which switches the word order...
+	//sometimes a word might not show up too...
+	//it really randomizes the band names, though
+	$("#name").click(function () {
+		$.get("adjective", function (response) {
+			$("#band_name").text(response.word);
+		});
+		$.get("noun", function (response) {
+			$("#band_name").append(" " + response.word);
+		});
+		$.get("verb", function (response) {
+			$("#band_name").append(" " + response.word);
+		});
+	});
+
+	// 	$("#name").click(function() {		
```

Comment:
Remove commented-out code for production (e.g. when you're pushing to Heroku) and for turning in assignments. You can always keep another branch with your commented-out code in it.